src/Jwt.php

```diff
@@ -0,0 +1,119 @@
+<?php
+declare(strict_types = 1);
+/**
+ *
+ * @see https://github.com/laminas/laminas-authentication for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-authentication/blob/master/COPYRIGHT.md
+ * @license https://github.com/laminas/laminas-authentication/blob/master/LICENSE.md New BSD License
+ */
+namespace Laminas\Authentication;
+
+class Jwt
```

Comment:
Can you explain why this class is needed? The authentication adapter itself uses only the `getPayload` method.
Thanks in advance!